src/NDItk/multigroup/Metadata.hpp

```diff
@@ -14,7 +14,7 @@ namespace NDItk {
 namespace multigroup {
 
 /**
- *  @brief Metadata associated to a neutron NDI file
+ *  @brief Metadata associated to multigroup neutron and photon data
```

Comment:
associated with*

Reply:
That has been changed in fix/review I think, following the sme remark in another PR